cluster/examples/kubernetes/ceph/upgrade-from-v1.0-create.yaml

```diff
@@ -4,13 +4,13 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: rook-ceph-cmd-reporter
-  namespace: rook-ceph
+  namespace: ROOK_NAMESPACE
```

Comment:
What if we used `%ROOK_NAMESPACE%` here? If the users try to use these manifests without the `sed` step, `kubectl` should complain about the invalid `%` character and expose the mistake early. I'm less concerned about potential name collisions, but that does seems like a small possibility.